tools/star_fusion_1.10.1_gen_reference.cwl

```diff
@@ -0,0 +1,58 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: star_fusion_1.10.1_gen_reference
+doc: "Generate star fusion reference. Recommend to run locally as tool will take at least 12 hours to run"
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/star:fusion-1.10.1'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    coresMin: $(inputs.cores)
+    ramMin: ${ return inputs.ram * 1000 }
+  - class: InitialWorkDirRequirement
+    listing: [$(inputs.ctat_source)]
+
+baseCommand: [tar, -I pigz, -xvf]
+arguments:
+  - position: 2
+    shellQuote: false
+    valueFrom: >-
+      && /STAR-Fusion/ctat-genome-lib-builder/prep_genome_lib.pl
+      ${
+          var arg = "--dfam_db ";
+          for (var i = 0; i < inputs.dfam_db.length; i++) {
+              if (inputs.dfam_db[i].nameext=='.hmm'){
+                  arg += inputs.dfam_db[i].path;
+                  return arg;
+              }
```

Comment:
Do you expect files in this file list that do not have an hmm extension?

Reply:
I do, but in the future I might hide this tool...